src/index.jsx

```diff
@@ -34,20 +35,27 @@ const App = () => (
               authenticatedAPIClient={apiClient}
               redirect={`${process.env.BASE_URL}`}
             />
+            <PrivateRoute
```

Comment:
Any reason this was moved?

Reply:
While resolving a merge conflict, my addition seemed to be pushing it further from it's related `new` route, so I moved them closer together.